resources.md

```diff
@@ -21,6 +21,7 @@ so that we can add them here.
  * [Java Bean Validation Extension](https://github.com/nomemory/java-bean-validation-extension) offers a set of useful constraints (`@Alphanumeric`, `@IPv6`, `@StartsWith`...).
  * [The validation project of Markus Malkusch](https://github.com/malkusch/validation) provides another set of constraints (`@BitcoinAddress`, `@ISBN`...).
  * [A nice example of password validator](https://github.com/Baeldung/spring-security-registration/blob/master/src/main/java/org/baeldung/validation/PasswordConstraintValidator.java) based on [Passay](http://www.passay.org/), a password policy enforcement library.
+ * The library [Lib-Validation](https://github.com/Naoghuman/lib-validation) from Naoghuman (Peter Rogge) is developed with the goal to allowed easily `validating` in a JavaFX application. See for example [@NewDuration](https://github.com/Naoghuman/lib-validation/blob/master/src/main/java/com/github/naoghuman/lib/validation/core/annotation/NewDuration.java). 
```

Comment:
If the library is doing more than just defining constraints, you might say:

> The library [Lib-Validation](https://github.com/Naoghuman/lib-validation) from Peter Rogge is developed with the goal to provide validation functionality for JavaFX applications. It also provides custom constraints, e.g. [@NewDuration](https://github.com/Naoghuman/lib-validation/blob/master/src/main/java/com/github/naoghuman/lib/validation/core/annotation/NewDuration.java).

Reply:
That's fine. I was also not sure if I should use my 'nickname' from my GitHub account, but your suggestion sound good (I think I will use it for my project description also :) ).

So I do must now a new PR for your suggestion?

Reply:
I wouldn't use the nickname if I were you, just add a link on your name to your GitHub profile.

You can make your changes directly in this PR, amend your commit and force push your branch.

HTH.

Reply:
No, you just can push another commit with this change to this PR. I'll then squash the two commits into a single one and merge it.